sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleMaterial.java

```diff
@@ -41,6 +41,11 @@
 	ENDOTRACHEAL_ASPIRATE,
 	BRONCHOALVEOLAR_LAVAGE,
 	BRAIN_TISSUE,
+	ANTERIOR_NARES_SWAB,
+	OROPHARYNGEAL_ASPIRATE,
+	NASOPHARYNGEAL_ASPIRATE,
+	LOWER_RESPIRATORY_SAMPLE,
```

Comment:
I added the values in the mapping excel here in the enum
![image](https://user-images.githubusercontent.com/65336966/112287750-5ccebb00-8c95-11eb-9656-d0bacd75bb3f.png)

I am not qualified to say whether it can be distinguished or how.

Reply:
If I understand the (full) excel table correctly, there shall not be an enum for lower respiratory sample in SORMAS. It shall be mapped to SampleMaterial.OTHER instead, whith DEMIS_DISPLAY mapped to Sample.SampleMaterialText.